private/buf/bufprint/bufprint.go

```diff
@@ -199,8 +199,8 @@ func PrintEntity(writer io.Writer, format Format, entity Entity) error {
 	}
 }
 
-// NewLabelEntity returns a new label entity to print.
-func NewLabelEntity(label *modulev1.Label, moduleFullName bufparse.FullName) Entity {
+// NewModuleLabelEntity returns a new label entity to print.
+func NewModuleLabelEntity(label *modulev1.Label, moduleFullName bufparse.FullName) Entity {
```

Comment:
I know we discussed having something like:

```
func NewLabelEntity(label interface{ GetName() string... etc. }, entityFullName bufparse.FullName)
```

I think this makes sense for both commit and labels to combine module and plugin.

Reply:
Switched to common interfaces for entities, same in #3499